UQ-eLIPSE/crucible-components
# story/VETUB-27

### I should be able to navigate quiz questions stack

To test:

- yarn run dev
- Navigate to http://localhost:5173/

Verify:

- Navigating the questions is enabled either by submitting or skipping.

Run tests:

- yarn run test
